cypress/integration/add_record.ts

```diff
@@ -0,0 +1,57 @@
+import { AuthRoles } from '../support/commands';
+
+describe('Adding records', () => {
+  describe('As a user in the Adults group', () => {
+    it('should go to the add case note form when "Case Note Recording" is selected', () => {
+      cy.visitAs(
+        `/people/${Cypress.env('ADULT_RECORD_PERSON_ID')}`,
+        AuthRoles.AdultsGroup
+      );
+
+      cy.contains('ASC');
+      cy.contains('Add a new record').click();
```

Comment:
Does this mean we can test for a component's presence (`contains`) and then interact with it (`click`)?
i.e. It's doing an `assertion` and then afterwards an `act` on the same line. 

Or is this a clever refactor and combining what could be multiple lines? 😁

Reply:
That's exactly right, Cypress methods are chainable, so it's getting the element that says "Add a new record" then clicking that element.

I can't take credit for this one though – I haven't actually changed these tests, I just moved the files around